package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mx-react-components",
-  "version": "6.5.1",
+  "version": "6.5.2",
```

Comment:
I think semantically speaking, this isn't a bug fix, but a new feature, so `6.6.0` would be slightly more appropriate.

Reply:
Updated